README.md

```diff
@@ -1,9 +1,9 @@
 # FINALE
 
-FINALE is a lightweight web app to view GCode. FINALE stands for FINALE Is Not A LabVIEW Editor.
+FINALE is a lightweight WebApp to view G Code. FINALE stands for FINALE Is Not A LabVIEW Editor.
```

Comment:
Avoid using the term "G Code". Maybe we could just say "LabVIEW code" or "code written using the G language"?

From [LabVIEW wiki](https://en.wikipedia.org/wiki/LabVIEW): "The graphical language is named "G"; not to be confused with G-code."

Reply:
We might also want to link to the LabVIEW page on ni.com. There could be people who land on this page without knowledge of LabVIEW, it'll generally help to setup some context.

Reply:
Changed to LabVIEW code, and addded a link to the LabVIEW page.